packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -311,13 +322,29 @@ export class QuoteRequestor {
         return true;
     }
 
-    private _makerSupportsPair(makerUrl: string, makerToken: string, takerToken: string): boolean {
-        for (const assetPair of this._rfqtAssetOfferings[makerUrl]) {
-            if (
-                (assetPair[0] === makerToken && assetPair[1] === takerToken) ||
-                (assetPair[0] === takerToken && assetPair[1] === makerToken)
-            ) {
-                return true;
+    private _makerSupportsPair(
+        typedMakerUrl: TypedMakerUrl,
+        makerToken: string,
+        takerToken: string,
+        altMakerAssetOfferings: AltRfqtMakerAssetOfferings | undefined,
+    ): boolean {
+        if (typedMakerUrl.pairType === 'standard') {
```

Comment:
I'd recommend turning this into an enum for consistent reference and the ability to lookup usage

Reply:
👍